inventory/group_vars/zuul

```diff
@@ -16,4 +20,8 @@ bonnyci_zuul_apache_vhosts:
       AliasMatch ^/p/(.*/objects/pack/pack-[0-9a-f]{40}.(pack|idx))$ {{ bonnyci_zuul_merger_git_dir }}/$1
       ScriptAlias /p/ /usr/lib/git-core/git-http-backend/
 
+      RewriteEngine on
+      RewriteRule ^/status.json$ http://127.0.0.1:8001/status.json [P]
+      RewriteRule ^/status/(.*) http://127.0.0.1:8001/status/$1 [P]
+
```

Comment:
I've created BonnyCI/projman#99 to address moving the merger to a dedicated host.